src/main/scala/is/hail/io/LoadMatrix.scala

```diff
@@ -337,8 +338,7 @@ object LoadMatrix {
     new MatrixTable(hc,
       matrixType,
       MatrixLocalValue(Annotation.empty,
-        sampleIds,
-        Annotation.emptyIndexedSeq(sampleIds.length)),
+        sampleIds.map(x => Annotation(x))),
```

Comment:
Can we do this where x is produced?  I realize this was an easy fix, but that's much more natural.  I'll have this comment in a few spots.

Reply:
this is actually a bit more complicated, since it'll involve fixing the logic that assumes the thing isn't a struct. OK to fix later?